src/components/NcCheckboxRadioSwitch/NcCheckboxContent.vue

```diff
@@ -9,6 +9,7 @@
 			['checkbox-content-' + type]: true,
 			'checkbox-content--button-variant': buttonVariant,
 			'checkbox-content--has-text': !!$slots.default,
+			'checkbox-content-switch-end': !isLegacy && type === 'switch',
```

Comment:
proper BEM

```suggestion
			'checkbox-content--switch-end': !isLegacy && type === 'switch',
```

Or maybe just like this?

```suggestion
			'checkbox-content--reverse': reverse,
```

Reply:
`reverse` could be a prop and then just injected in the main component of the radio switch